syntax = "proto3";

option go_package = "../common";

message WeatherData {
  string state = 1;
  string station_name = 2;
  string temp_f = 3;
  string weather = 4;
  string last_update = 5;
  string station_id = 6;
  string latitude = 7;
  string longitude = 8;
  string windchill_f = 9;
  string relative_humidity = 10;
  string wind_speed = 11;
  string visibility = 12;
}